Add New Zealand admin source

stepps00 opened the pull request:
Fixes https://github.com/whosonfirst/whosonfirst-sources/issues/179

Adds a new source json file for the Land Information New Zealand site and runs the makefile to update spec docs/readme.

nvkelso commented on sources/nz-linz.json:
```diff
@@ -0,0 +1,16 @@
+    "fullname": "Land Information New Zealand", 
+    "id": 1729181221, 
+    "key": "", 
+    "license": "https://data.linz.govt.nz/layer/104830-fire-and-emergency-nz-localities/", 
```
Is there a generic license for all LINZ data? This license text is specific. So Either make the source full anme and prefix specific, or generalize the license text?

nvkelso reviewed (approved):
A few questions, but otherwise LG.

stepps00 commented on sources/nz-linz.json:
```diff
@@ -0,0 +1,16 @@
+    "name": "nz-linz", 
+    "prefix": "nz-linz", 
+    "url": "https://data.linz.govt.nz/", 
+    "usage_concordance": 0, 
```
changed to 1 in most recent commit.

stepps00 commented:
In most recent commits, I:

- Updated the license text to point to the generic site license
- Updated the concordance usage flag
- Update the url to point to the data source page, rather than the main source page

(This is similar to how we handle similar license files.. [example](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/aus-psma.json))

nvkelso commented:
🚀
